components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/util/SAMLSSOUtil.java

```diff
@@ -754,28 +744,28 @@ public static ArtifactResponse setSignature(ArtifactResponse request, String sig
     private static SignableXMLObject doSetSignature(SignableXMLObject request, String signatureAlgorithm, String
             digestAlgorithm, X509Credential cred) throws IdentityException {
 
-            doBootstrap();
-            try {
-                synchronized (Runtime.getRuntime().getClass()) {
-                    ssoSigner = (SSOSigner) Class.forName(IdentityUtil.getProperty(
-                            SAMLSSOConstants.SAMLSSO_SIGNER_CLASS_NAME).trim()).newInstance();
-                    ssoSigner.init();
-                }
+        doBootstrap();
+        try {
+            synchronized (Runtime.getRuntime().getClass()) {
+                ssoSigner = (SSOSigner) Class.forName(IdentityUtil.getProperty(
+                        SAMLSSOConstants.SAMLSSO_SIGNER_CLASS_NAME).trim()).newInstance();
+                ssoSigner.init();
+            }
```

Comment:
Create a new issue to move this to bootstrap

Reply:
Created issue https://github.com/wso2-extensions/identity-inbound-auth-saml/issues/263